Assets/Scripts/NpcBehaviors/BaseNpc.cs

```diff
@@ -133,6 +136,21 @@ protected void Start()
     /// </summary>
     protected virtual void Initialize()
     {
+        GameObject tempObject = GameObject.FindGameObjectWithTag("Inventory");
```

Comment:
Is this the NPC's inventory? If that's the case you could probably put it on the NPC itself and grab that via GetComponent